sky/clouds/gcp.py

```diff
@@ -547,7 +542,7 @@ def make_deploy_resources_variables(
                         acc.lower())
                 resources_vars['gpu_count'] = acc_count
                 if enable_gpu_direct or network_tier == resources_utils.NetworkTier.BEST:
-                    image_id = _NETWORK_GCP_IMAGE_ID
+                    image_id = _DEFAULT_GPU_DIRECT_IMAGE_ID
```

Comment:
Add a comment here saying the actual image used would be a docker image and link to the place we set it to a docker image.